lms/views/error.py

```diff
@@ -48,3 +49,19 @@ def error(self):
         return {'message': _("Sorry, but something went wrong. "
                              "The issue has been reported and we'll try to "
                              "fix it.")}
+
+
+    @view_config(context=MissingLTILaunchParamError)
+    def missing_lti_launch_param_error(self):
+        """
+        Raise MissingLTILaunchParamError if params required for lti launch are missing.
```

Comment:
This method doesn't _raise_ `MissingLTILaunchParamError`s, it catches them and renders a 400 page containing the exception's message, and reports the exception to Sentry (as you've said in the rest of the docstring below)